src/test/java/juangonzales_reqres/ParameterizationTest.java

```diff
@@ -0,0 +1,42 @@
+package juangonzales_reqres;
+
+
+import static io.restassured.RestAssured.given;
+import static org.hamcrest.Matchers.equalTo;
+
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import com.tngtech.java.junit.dataprovider.DataProvider;
+import com.tngtech.java.junit.dataprovider.DataProviderRunner;
+import com.tngtech.java.junit.dataprovider.UseDataProvider;
+
+@RunWith(DataProviderRunner.class)
+public class ParameterizationTest {
+
+
+    @DataProvider
+    public static Object[][] dataProviderAdd() {
+        return  new Object[][]{
+                {"us", "90210", "Beverly Hills"},
+                {"us", "12345", "Schenectady"},
+                {"ca", "B2R", "Waverley"},
+                {"nl", "1001", "Amsterdam"}
+        };
+    }
+
+    @Test
+    @UseDataProvider("dataProviderAdd")
+    public void testAdd(String countryCode, String zipCode, String expectedResult)  {
```

Comment:
it looks good, but it does not run. Just in case I changed the dependencies in build.gradle